roachtest: pgjdbc failed

roachtest.pgjdbc [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7377876?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7377876?buildTab=artifacts#/pgjdbc) on release-22.2 @ [9048c36c4f4e851bfabde9209083fadeee7aa532](https://github.com/cockroachdb/cockroach/commits/9048c36c4f4e851bfabde9209083fadeee7aa532):


```
		  | FAILURE: Build failed with an exception.
		  |
		  | * What went wrong:
		  | A problem occurred configuring root project 'pgjdbc'.
		  | > Could not resolve all artifacts for configuration ':classpath'.
		  |    > Could not find org.ajoberstar.grgit:grgit-core:4.0.1.
		  |      Searched in the following locations:
		  |        - https://plugins.gradle.org/m2/org/ajoberstar/grgit/grgit-core/4.0.1/grgit-core-4.0.1.pom
		  |      If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
		  |      Required by:
		  |          project : > com.github.vlsi.stage-vote-release:com.github.vlsi.stage-vote-release.gradle.plugin:1.71 > com.github.vlsi.gradle:stage-vote-release-plugin:1.71
		  |          project : > com.github.vlsi.stage-vote-release:com.github.vlsi.stage-vote-release.gradle.plugin:1.71 > com.github.vlsi.gradle:stage-vote-release-plugin:1.71 > org.ajoberstar.grgit:grgit-gradle:4.0.1
		  |
		  | * Try:
		  | Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
		  |
		  | * Get more help at https://help.gradle.org
		  |
		  | BUILD FAILED in 1s
		  |
		  | stdout:
		  | > Task :buildSrc:compileKotlin UP-TO-DATE
		  | > Task :buildSrc:compileJava NO-SOURCE
		  | > Task :buildSrc:compileGroovy NO-SOURCE
		  | > Task :buildSrc:pluginDescriptors UP-TO-DATE
		  | > Task :buildSrc:processResources NO-SOURCE
		  | > Task :buildSrc:classes UP-TO-DATE
		  | > Task :buildSrc:inspectClassesForKotlinIC UP-TO-DATE
		  | > Task :buildSrc:jar UP-TO-DATE
		  | > Task :buildSrc:assemble UP-TO-DATE
		  | > Task :buildSrc:autostyleKotlinCheck UP-TO-DATE
		  | > Task :buildSrc:autostyleKotlinGradleCheck UP-TO-DATE
		  | > Task :buildSrc:autostyleCheck UP-TO-DATE
		  | > Task :buildSrc:compileTestKotlin NO-SOURCE
		  | > Task :buildSrc:pluginUnderTestMetadata UP-TO-DATE
		  | > Task :buildSrc:compileTestJava NO-SOURCE
		  | > Task :buildSrc:compileTestGroovy NO-SOURCE
		  | > Task :buildSrc:processTestResources NO-SOURCE
		  | > Task :buildSrc:testClasses UP-TO-DATE
		  | > Task :buildSrc:test NO-SOURCE
		  | > Task :buildSrc:validatePlugins UP-TO-DATE
		  | > Task :buildSrc:check UP-TO-DATE
		  | > Task :buildSrc:build UP-TO-DATE
		Wraps: (6) COMMAND_PROBLEM
		Wraps: (7) Node 1. Command with error:
		  | ``````
		  | cd /mnt/data1/pgjdbc/pgjdbc/ && ../gradlew test --tests OidToStringTest
		  | ``````
		Wraps: (8) exit status 1
		Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *cluster.WithCommandDetails (6) errors.Cmd (7) *hintdetail.withDetail (8) *exec.ExitError
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91369 roachtest: pgjdbc failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91183 roachtest: pgjdbc failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #90232 roachtest: pgjdbc failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*pgjdbc.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-21291

--- a/pkg/cmd/roachtest/tests/pgjdbc_blocklist.go
+++ b/pkg/cmd/roachtest/tests/pgjdbc_blocklist.go
@@ -481,7 +481,6 @@ var pgjdbcBlockList = blocklist{
 	"org.postgresql.test.jdbc2.ServerErrorTest.testPrimaryKey":                                                                                                                 "27796",
 	"org.postgresql.test.jdbc2.StatementTest.testCloseInProgressStatement":                                                                                                     "17511",
 	"org.postgresql.test.jdbc2.StatementTest.testConcurrentWarningReadAndClear":                                                                                                "17511",
-	"org.postgresql.test.jdbc2.StatementTest.testDateFunctions":                                                                                                                "17511",
 	"org.postgresql.test.jdbc2.StatementTest.testFastCloses":                                                                                                                   "17511",
 	"org.postgresql.test.jdbc2.StatementTest.testParsingSemiColons":                                                                                                            "17511",
 	"org.postgresql.test.jdbc2.StatementTest.testUpdateCount":                                                                                                                  "17511",

